client/components/marketing-survey/cancel-purchase/index.jsx

```diff
@@ -0,0 +1,357 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import shuffle from 'lodash/shuffle';
+
+/**
+ * Internal Dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+
+const CancelPurchaseSurvey = React.createClass( {
+	propTypes: {
+		surveyStep: React.PropTypes.number.isRequired,
+		showSurvey: React.PropTypes.bool.isRequired,
+		defaultContent: React.PropTypes.node.isRequired,
+		callbackParent: React.PropTypes.func.isRequired
```

Comment:
`callbackParent` is pretty generic. We're calling this whenever an input changes, right? Could we just call this `onInputChange` so it's a little more clear?